examples/transfers/cancellations_test.go

```diff
@@ -0,0 +1,53 @@
+package transfers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/moovfinancial/moov-go/pkg/moov"
+)
+
+func ExampleClient_CancelTransfer() {
```

Comment:
wondering if the example should start from creating the transfer and then to cancelling it

Reply:
We should be good to not create a transfer first -- and keep this example shorter. An example for creating a transfer should go in its own example func